real_estate_module/models/estate_property.py

```diff
@@ -0,0 +1,26 @@
+from odoo import models, fields
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate Property"
+
+    name = fields.Char(required=True)              # Property name
```

Comment:
```suggestion
    name = fields.Char(required=True)
```
Don't you think you can make use of [common attributes](https://www.odoo.com/documentation/18.0/developer/tutorials/server_framework_101/03_basicmodel.html?#common-attributes) ? Which are user friendly and developer friendly too